### PR TITLE
Update Es.php

### PR DESCRIPTION
**Pull request by xeux**
Support for Panamanian Balboa (PAB)

### CODE REVIEW

File: src/Legacy/Numbers/Words/Locale/Es.php [hunk 1]
```diff
@@ -77,6 +77,7 @@ class Es extends Words
         'MTL' => [['lira maltesa'], ['céntimo']],
         'MXN' => [['peso'], ['centavo']],
         'NOK' => [['norwegian krone'], ['oere']],
+        'PAB' => [['balboa', 'balboas'], ['centavo', 'centavos']],
```
Comment (kwn) [inline src/Legacy/Numbers/Words/Locale/Es.php]: All other currencies have only one version of subcurrency. Are you sure two versions (`'centavo', 'centavos'`) will be handled correctly?

Comment (xeux) [reply src/Legacy/Numbers/Words/Locale/Es.php]: I have not tested myself, but for what I understand that array is [singular, plural], not sure why other currencies as MXN which is also from a Spanish speaking country is not handling plural.

Comment (xeux) [reply src/Legacy/Numbers/Words/Locale/Es.php]: Also, NOK text is in English in a Spanish file, look at Wikipedia. [https://es.wikipedia.org/wiki/Corona_noruega](url)

**Comment by xeux** [comment]
I just commit two test for the PAB currency, let me know if you need something else.

**Comment by kwn** [comment]
Hey @xeux 

Thank you for adding tests. Could you push tests to the `xeux:patch-1` branch so CI can verify if the PR (change in the code and tests) work fine? Thanks!

**Comment by kwn** [comment]
Hey @xeux 

Any chance you can merge branch `xeux:patch-1` and `xeux:patch-2` and open a new PR with both of them?

**Comment by xeux** [comment]
@kwn is there any way to do it from github web?

**Comment by kwn** [comment]
@xeux
I don't think so. Most likely you'll need to merge branches in your local repo and push them to your fork.

**Comment by cluis13915** [comment]
Hey! I also would like to add support for GTQ (Quetzales from Guatemala). How can I do that ? I can see this hasn't been merged since 2019 :confused: 

```
private static $currencyNames = [
    ...,
    'GTQ' => [['quetzal', 'quetzales'], ['centavo', 'centavos']],
];

**Pull request merged**